samples/history/basic.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "1.1.0",
+  "startDate": "2018-10-06T04:00:00Z",
+  "endDate": "2018-10-06T04:00:02Z",
+  "generator": "Hand-written with love with the spec",
+
+  "objects": [
+    {
+      "context": "vessels.urn:mrn:xxx",
+      "timestamps": [ "2018-10-06T04:00:00Z", "2018-10-06T04:01:00Z", "2018-10-06T04:00:02Z"],
+      "properties": [
+        {
+          "path": "navigation.position",
+          "source": { "label": "NMEA1" },
+          "values": [
+            { "longitude": -182.2, "latitude": -42.1},
+            { "longitude": -182.2, "latitude": -42.1},
+            { "longitude": -182.2, "latitude": -42.1}
+          ]
+        },
+        {
+          "path": "navigation.speedOverGround",
+          "source": { "label": "NMEA1" },
```

Comment:
Alternatively the single string, dot notation.

Reply:
I am not following. Can you give an example?